From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

WilliamAntonRohm committed:
Merge pull request #5 from MicrosoftDocs/master

bring 'em in

WilliamAntonRohm committed:
Merge pull request #6 from MicrosoftDocs/master

bring in masterful changes

WilliamAntonRohm committed:
Merge pull request #11 from MicrosoftDocs/release-ignite-aml-v2

Release ignite aml v2

WilliamAntonRohm committed:
Merge pull request #12 from MicrosoftDocs/master

br

WilliamAntonRohm committed:
Merge pull request #14 from MicrosoftDocs/master

b

WilliamAntonRohm committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr
sdf
ljasdf
lkjasdf

BryanLa committed:
initial add

rloutlaw committed:
add devcenter metadata to content linked from quickstart tiles on language devcenters

BryanLa committed:
updates

sethmanheim committed:
Adding article on messages and payloads

sethmanheim committed:
Acrolinx

BryanLa committed:
updates

sethmanheim committed:
Adding some links

Commit:
fixing rg and wrapping

sethmanheim committed:
Adding some links

sethmanheim committed:
Adding to TOC

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into bryanla-msi-token-sp

BryanLa committed:
updates

sethmanheim committed:
Adding new article on message transfers

sethmanheim committed:
Acrolinx

sethmanheim committed:
Adding to TOC

sethmanheim committed:
A few fixes

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cv1

sethmanheim committed:
Removing placeholder links

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cv2

sethmanheim committed:
Removing tbd links

BryanLa committed:
initial add

BryanLa committed:
initial add

BryanLa committed:
initial add

BryanLa committed:
initial add

BryanLa committed:
initial add

sanjeev3 committed:
Create docs

sanjeev3 committed:
Delete docs

sanjeev3 committed:
Create test.md

sanjeev3 committed:
Delete test.md

sanjeev3 committed:
Create test

sethmanheim committed:
Adding new article on message browsing

v-vikum2 committed:
USER STORY 1088515 SaaS App Tutorial: XaitPorter

shruja committed:
USER STORY 1088581: SaaS App Tutorial: Compliance ELF

v-vikum2 committed:
USER STORY 1088545 SaaS App Tutorial: SmartRecruiters

sethmanheim committed:
Updates

sethmanheim committed:
Add to TOC

WilliamAntonRohm committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ljasdf
ljasdf
he commit.

Commit:
safekeeping

WilliamAntonRohm committed:
Merge pull request #16 from MicrosoftDocs/master

bring them in

Commit:
Adding error 40914, per PM Dhruv Malik. MightyPen = genemi.

sethmanheim committed:
Clean up

kgremban committed:
Integrate changes from PR 1825 for IoT edge

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

sethmanheim committed:
More link updates

BryanLa committed:
updates

sethmanheim committed:
Adding article on message sequencing

sethmanheim committed:
Adding to TOC

rwike77 committed:
merged content

rwike77 committed:
updated redirects and toc

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into javaactor

rwike77 committed:
removed article

shruja committed:
metadata url change

shruja committed:
USER STORY 1087630: SaaS App Tutorial: Expensify Update

v-vikum2 committed:
done

sanjeev3 committed:
Delete test

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Create ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

bsiva committed:
Adding borders to pictures

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Delete tutorial-computer-vision-keys.PNG

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Delete tutorial-ecommerce-tags2.PNG

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

ggailey777 committed:
Remove anchor link from the TOC

This Carol-approved experiment failed due to duplicate topics being included in the generated PDF. Reverting back to a known good state for the TOC.

Juliako committed:
Update media-services-frequently-asked-questions.md

spelluru committed:
changed the date. demo

BryanLa committed:
updates

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into sept28_pr1825

sanjeev3 committed:
Update ecommerce-retail-catalog-moderation.md

Lisaco88 committed:
Merge pull request #24133 from sethmanheim/cv2

Adding new article on message transfers

PRMerger20 committed:
Merge pull request #24461 from spelluru/adfintrodate

changed the date. demo

curtand committed:
removed reference to iOS, which does not work

rmca14 committed:
Merge pull request #24456 from ggailey777/patch-1

Remove anchor links from the TOC

hning86 committed:
Update tutorial-classifying-iris-part-2.md

rmca14 committed:
Merge pull request #24044 from sethmanheim/cv1

Adding article on messages and payloads

spelluru committed:
on-demand is supproted now

mahesh-unnikrishnan committed:
Delete active-directory-ds-azure-portal-enable-preview-access.md

PRMerger20 committed:
Merge pull request #24367 from kgremban/sept28_pr1825

Integrate changes from PR 1825 for IoT edge

mahesh-unnikrishnan committed:
Delete missing-domain-services-instance.png

markgalioto committed:
linking the Backup CLI reference in tutorial-restore-files

mahesh-unnikrishnan committed:
Update .openpublishing.redirection.json

PRMerger12 committed:
Merge pull request #24466 from curtand/jarrett0929

replaced reference to iOS, which does not work

ErikjeMS committed:
tenant->user

mahesh-unnikrishnan committed:
Update TOC.md

rmca14 committed:
Merge pull request #24407 from rwike77/javaactor

Merging Java actor articles

v-shils committed:
Merge pull request #24433 from bsiva/master

Adding borders to pictures

ErikjeMS committed:
tenant->user

kgremban committed:
Freshness, and incorporating public PR 1991

kgremban committed:
Freshness, and incorporate changes from public pr 1992

PRMerger19 committed:
Merge pull request #24477 from kgremban/sept29_pr1991

Freshness, and incorporating public PR 1991

v-shils committed:
Merge pull request #24305 from v-vikum2/xaitporterv1

USER STORY 1088515 SaaS App Tutorial: XaitPorter

dubansal committed:
Fixing typo

PRMerger13 committed:
Merge pull request #24470 from spelluru/adfspark0929

on-demand is supproted now

Commit:
Merge pull request #24479 from kgremban/sept29_pr1992

Freshness, and incorporate changes from public pr 1992

PRMerger10 committed:
Merge pull request #24458 from Juliako/patch-1

Update media-services-frequently-asked-questions.md

PRMerger12 committed:
Merge pull request #24476 from ErikjeMS/master

Tenant->user

btardif committed:
fixing documentation to match implementation

btardif committed:
Merge remote-tracking branch 'azure/master'

rwike77 committed:
refreshed

Lisaco88 committed:
Merge pull request #24424 from sanjeev3/master

New Tutorial on ecommerce catalog moderation

WilliamAntonRohm committed:
Merge pull request #17 from MicrosoftDocs/master

bring them in

PRMerger19 committed:
Merge pull request #24471 from markgalioto/AB-9-29

linking the Backup CLI reference in tutorial-restore-files

v-shils committed:
Merge pull request #24418 from shruja/expensify

USER STORY 1087630: SaaS App Tutorial: Expensify Update

v-shils committed:
Merge pull request #24308 from v-vikum2/smartrecruitersv1

USER STORY 1088545 SaaS App Tutorial: SmartRecruiters

Lisaco88 committed:
Merge pull request #24240 from BryanLa/bryanla-msi-config-updates

MSI - add config via SDKs, howto sign-in and acquire access token

Ja-Dunn committed:
Merge pull request #24484 from rloutlaw/dev-center-metadata

add devcenter metadata to content linked from quickstart tiles on lan…

Commit:
log files

PRMerger18 committed:
Merge pull request #24467 from hning86/patch-9

Update tutorial-classifying-iris-part-2.md, minor format update.

Commit:
Merge pull request #24483 from rwike77/freshness

refreshed

Ja-Dunn committed:
Merge pull request #24469 from mahesh-unnikrishnan/patch-15

Delete active-directory-ds-azure-portal-enable-preview-access.md

sethmanheim committed:
Fixing merge conflict

Ja-Dunn committed:
Merge pull request #24306 from shruja/complianceelf

USER STORY 1088581: SaaS App Tutorial: Compliance ELF

Ja-Dunn committed:
Merge pull request #24163 from cynthn/92517-2

fixing rg and wrapping

PRMerger20 committed:
Merge pull request #24482 from btardif/master

Fixing doc bug for App Service IP restrictions

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cv5

Ja-Dunn committed:
Merge pull request #24488 from hning86/logs

adding log file locations

Commit:
freshness

sethmanheim committed:
Fixing merge conflict

spelluru committed:
Data Factory V2 limits

rwike77 committed:
freshness updates

spelluru committed:
fixed broken link

WilliamAntonRohm committed:
updating author so I can delete this file next round

sethmanheim committed:
Adding another H2

Commit:
Merge pull request #24491 from cynthn/master

Connect

spelluru committed:
acrolynx

Commit:
Merge pull request #24493 from rwike77/freshness

freshness updates

Commit:
2. Adding subsection for error 40615, per Dhruv.

Commit:
Merge pull request #24494 from WilliamAntonRohm/W-Jennie-redis

updating the author so I can delete this on the next go-round

Commit:
Merge pull request #24480 from dubansal/patch-1

Fixing typo

Ja-Dunn committed:
Merge pull request #24282 from sethmanheim/cv5

Adding new article on message browsing

WilliamAntonRohm committed:
Merge pull request #18 from MicrosoftDocs/master

bring it in

PRMerger17 committed:
Merge pull request #24365 from MightyPen/vnetfixprodanitha-20170926-a

Adding error 40914, per PM Dhruv Malik.

sethmanheim committed:
Merge branch 'master' into cv7

cephalin committed:
livefyre feedback

Ja-Dunn committed:
Merge pull request #24492 from spelluru/adflimits0929

Data Factory V2 limits

cephalin committed:
tweak

WilliamAntonRohm committed:
removing stream-analytics-functions-redis & its TOC entry & updating redirection

Ja-Dunn committed:
Merge pull request #24496 from WilliamAntonRohm/W-Jennie-redis-delete

removing stream-analytics-functions-redis.md & its TOC entry; updating redirection

Ja-Dunn committed:
Merge pull request #24395 from sethmanheim/cv7

Adding article on message sequencing

PRMerger20 committed:
Merge pull request #24495 from cephalin/customimage

Customimage

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #24498 from MicrosoftDocs/FromPublicRepo

From public repo
